### DISCUSSION TITLE
Get info about the latest version of a package

### DISCUSSION THREAD

**Discussion by Shayan-To** [yarnpkg/berry#3944] [Q&A] [answered]
How can I get info about the latest version of a package? It seems like `yarn info package@latest` will try to find it locally and returns a no-match error.

**Comment by paul-soporan** [comment, marked as answer]
If it's a package from the npm registry you can run `yarn npm info package` (`latest` is implied and can't be currently specified as a range but I'll work on fixing that).

**Reply by Shayan-To** [reply]
Thanks for the answer.

I actually want to get info about `vue@next`. If I understand correctly, it is not currecntly possible, right?

**Reply by paul-soporan** [reply]
It is not, but I've just implemented support for it a few minutes ago in https://github.com/yarnpkg/berry/pull/3987.

**Reply by Shayan-To** [reply]
Thanks.
